sdk/communication/tests.yml

```diff
@@ -0,0 +1,18 @@
+trigger: none
+
+jobs:
+  template: ../../eng/pipelines/templates/jobs/archetype-sdk-tests.yml
+  parameters:
+    Artifacts:
+      - name: azure-communication-administration
+        groupId: com.azure
+        safeName: azurecommunicationadministration
+      - name: azure-communication-chat
+        groupId: com.azure
+        safeName: azurecommunicationchatclient
+      - name: azure-communication-sms
+        groupId: com.azure
+        safeName: azurecommunicationsms
+    ResourceServiceDirectory: communication
+    EnvVars:
```

Comment:
If you want to eventually pass in other secrets for things that are static resources you might need to add some more secrets and pass them similar to https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/keyvault/tests.yml#L21.

Reply:
Thanks Wes, we've decided to hold off on using static resources for now until there is a better long-term solution.  Live SMS tests will not be part of automation for now.